src/main/java/uk/gov/companieshouse/company/links/consumer/ChargesStreamConsumer.java

```diff
@@ -25,25 +25,15 @@ public ChargesStreamConsumer(ChargesStreamProcessor streamProcessor) {
     /**
      * Receives Main topic messages.
      */
-    @KafkaListener(topics = "${charges.stream.topic.main}", groupId = "charges.stream.topic.main",
+    @KafkaListener(topics = "${company-links.consumer.charges.topic.main}", 
+            groupId = "${company-links.consumer.charges.group-id}",
             autoStartup = "${company-links.consumer.charges.enable}")
     @Retryable
```

Comment:
We should probably remove this @Retryable annotation

Reply:
Yep - removed